### ISSUE TITLE
binary issue

### ISSUE THREAD

**Issue by surya182827** [openntpd-portable/openntpd-portable#54]
Hi
i Used this repository to create binaries for both arm and AMD platform before  and i am able to generate the binaries successfully  but when i am trying to create the binaries now i am running into an error as below

Initial build output (August 2018)
 CC       ntpd-client.o
  CC       ntpd-config.o
  CC       ntpd-constraint-disabled.o
  CC       ntpd-control.o
  CC       ntpd-log.o
  CC       ntpd-ntp.o
  CC       ntpd-ntp_dns.o
  CC       ntpd-ntp_msg.o
  CC       ntpd-ntpd.o
  YACC     parse.c
  CC       ntpd-parse.o
  CC       ntpd-fake-sensors.o
  CC       ntpd-server.o
  CC       ntpd-util.o
  CCLD     ntpd

Most recent build output 

parse.y: In function 'yyparse':
parse.y:439:57: error: 'RT_TABLEID_MAX' undeclared (first use in this function)
    if ($2 < 0 || $2 > RT_TABLEID_MAX) {
                                                         ^             
parse.y:439:57: note: each undeclared identifier is reported only once for each function it appears in
Makefile:668: recipe for target 'ntpd-parse.o' failed

What changed with before and now

**Comment by busterb** [comment]
What OS are you targeting?

**Comment by surya182827** [comment]
Debian stretch
But the issue with binary was solved and i ran into a new issue . I just
posted it

Thanks
Surya
